labcodes/confetti-storybook
# [CSYS-366]  Migrating DropdownSectionTitle, TagDropdownItem and TagDropdown to Hooks

**Link to task:** [CSYS-366](https://labcodes.atlassian.net/browse/CSYS-366), [CSYS-367](https://labcodes.atlassian.net/browse/CSYS-366) and [CSYS-368](https://labcodes.atlassian.net/browse/CSYS-366)

**Staging app:** 
CSYS-366-migrate-dropdown-section-title--confetti-storybook.netlify.app/

**How to test:** 
After running your project locally, run the TagDropdownItem.test.js, TagDropdown.test.js and DropdownSectionTitle.test.js in order to check if the TagDropdown component is working as expected.

**Description of your solution:**
Removing previous mentions from Class Components and moving the Props outside the function and also memoized checkThumbAndIcon callback function.

Screenshots (when applicable): NA
